apps/core/components/userContext/SignInHelpers.native.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import { Google } from 'expo';
+import { Platform } from 'react-native';
+import firebase from 'firebase/app';
+import { GOOGLE_OAUTH_ANDROID, GOOGLE_OAUTH_IOS } from 'react-native-dotenv';
+
+/**
+ * NOTE: Direct Firebase login is not available for RN.
+ * We are using `expo` Goole login to obtain user token.
+ * And then passing it to Firebase `GoogleAuthProvider` to log user in.
+ */
+
+export const signInWithGoogle = () => {
+  return new Promise<void>((resolve, reject) => {
+    Google.logInAsync({
```

Comment:
it seems this already is returning a promise. Why not do 
`export const signInWithGoogle = async () => {`